std/http/server.ts

```diff
@@ -237,6 +237,30 @@ export class Server implements AsyncIterable<ServerRequest> {
 /** Options for creating an HTTP server. */
 export type HTTPOptions = Omit<Deno.ListenOptions, "transport">;
 
+/**
+ * Parse addr from string
+ *
+ *     const addr = "::1:8000";
+ *     parseAddrFromString(addr);
+ *
+ * @param addr Address string
+ */
+export function parseAddrFromStr(addr: string): HTTPOptions {
```

Comment:
This should be private, rename it to `_parseAddrFromStr()` if it must be exported.

Reply:
👍

Reply:
@nayeemrmn , pushed changes, how does it look to you?

Reply:
If all is good, I'd love to squash commits merge this PR